src/modules/sih/sih.cpp

```diff
@@ -429,9 +429,6 @@ void Sih::publish_sih()
 	_att_gt.q[1] = _q(1);
 	_att_gt.q[2] = _q(2);
 	_att_gt.q[3] = _q(3);
-	_att_gt.rollspeed = _w_B(0);
-	_att_gt.pitchspeed = _w_B(1);
-	_att_gt.yawspeed = _w_B(2);
```

Comment:
I suggest adding a subtopic `vehicle_angular_velocity_groundtruth` and publish it here.

Reply:
Added.